api/handlers.go

```diff
@@ -39,6 +40,19 @@ func getInfo(c *context, w http.ResponseWriter, r *http.Request) {
 		MemTotal:          c.cluster.TotalMemory(),
 	}
 
+	if httpProxy := os.Getenv("http_proxy"); httpProxy != "" {
```

Comment:
probably not, to be sincere I just copy pasted from docker/docker...

I'll update